### ISSUE TITLE
How to evaluate the performance of denoising when i Compared two weights?

### ISSUE THREAD

**Issue by wowowmmmmm** [xiph/rnnoise#148]
i trained two weights of the rnnoise.
when i tested the wav by myself, each ones all got a good denoising performance. 
However , i do not know which one is the best ?
so, do you know some ways to  evaluate the performance of denoising or the code to evaluate the performance of denoising wavs?

**Comment by jagger2048** [comment]
Hi,
  You can use the PESQ to evaluate the output of the two weights.
   [python-pesq](https://github.com/ludlows/python-pesq)

B&R

**Comment by wowowmmmmm** [comment]
ok,3q
however when i use pypesq ,  i had a problem of the usage;
the ref signal is the data before mixing noise or the data after mixing noise

**Comment by guishengzhang** [comment]
The ref signal should be clean speech before mixing noise.

**Comment by RXAldreezee** [comment]
> Hi,
> You can use the PESQ to evaluate the output of the two weights.
> [python-pesq](https://github.com/ludlows/python-pesq)
> 
> B&R

I agree with this. Make sure you have clean wav file and rnnoise-processed wav file. Produce a wav file in each of the model and use the python-pesq to compare. All wav files should have the same audio duration and 16k sampling frequency.
